src/remoteControlConfig.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Configures remote playback behavior.
+ */
+export interface RemoteControlConfig {
+  /**
+   * A URL to a CSS file the receiver app loads to style the receiver app.
+   * Default value is `null`, indicating that the default CSS of the receiver app will be used.
+   */
+  receiverStylesheetUrl?: string | null;
+  /**
+   * A Map containing custom configuration values that are sent to the remote control receiver.
+   * Default value is an empty map.
+   */
+  customReceiverConfig?: Record<string, string | null>;
+  /**
+   * Whether casting is enabled.
+   * Default value is `true`.
+   *
+   * Has no effect if the BitmovinCastManager is not initialized in the app's context.
```

Comment:
```suggestion
   * Has no effect if the `BitmovinCastManager` is not initialized in the app's context.
```
Does that make sense for iOS as well?

Reply:
Improved in 1d77a80